.drone.yml

```diff
@@ -127,6 +127,29 @@ pipeline:
       status: [ success, failure ]
       event: [push, pull_request]
 
+  docker-up:
+    image: docker/compose:1.17.1
+    commands:
+      - cd docker
+      - cp drone_env/* env_files/
+      - cp drone_env/.env .
+      # 1. start containers
+      - docker-compose stop
```

Comment:
Seems it is an unnecessary line

Reply:
@artemiy312 , without it we get "port is already allocated".
But seems i got more good decision to fix it